plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestingPostgreSqlServer.java

```diff
@@ -96,7 +102,11 @@ protected List<RemoteDatabaseEvent> getRemoteDatabaseEvents()
         while (logsIterator.hasNext()) {
             String logLine = logsIterator.next().replaceAll(LOG_PREFIX_REGEXP, "");
             if (logLine.startsWith(LOG_RUNNING_STATEMENT_PREFIX)) {
-                events.add(new RemoteDatabaseEvent(logLine.substring(LOG_RUNNING_STATEMENT_PREFIX.length()), RUNNING));
+                Matcher matcher = SQL_QUERY_FIND_PATTERN.matcher(logLine.substring(LOG_RUNNING_STATEMENT_PREFIX.length()));
+                if (matcher.find()) {
```

Comment:
Why condition?

Reply:
So it's all because in pgsql cursor and non-cursor queries are logged differently and previously we were not adding cursor queries to the events log. Here we are actually trying to find the queries and log the SQL, no matter the cursor used or not.